extension/src/experiments/columns/collect/metricsAndParams.ts

```diff
@@ -38,10 +38,16 @@ const collectMetricOrParam = (
 
   mergeValueColumn(
     acc,
-    buildMetricOrParamPath(type, ...limitedDepthAncestors, label),
+    buildMetricOrParamPath(
+      type,
+      ...(limitedDepthAncestors || pathArray),
+      label
+    ),
     buildMetricOrParamPath(type, ...limitedDepthAncestors),
     [type, ...pathArray, label],
-    label,
+    limitedDepthAncestors.length === 0
+      ? buildMetricOrParamPath(type, ...pathArray, label).slice(type.length + 1)
+      : label,
```

Comment:
[C] This call to `mergeValueColumn` is way too verbose for my tiny pea brain.

Reply:
Moved label creation to `limitAncestorDepth`!